csv-extractor/csv_extractor.py

```diff
@@ -100,29 +102,46 @@ def create_time_series(client, name: str, external_id: str) -> None:
     _log_error(client.time_series.post_time_series, [new_time_series])
 
 
+def get_parsed_file(path) -> Dict[str, str]:
+    """Parse the csv file and return the data in dictionary"""
+    parsed_file = {}
+    with open(path, "r", encoding="latin-1") as f:
+        data = csv.DictReader(f, delimiter=";")
+        for row in data:
+            for (k, v) in row.items():
+                try:
```

Comment:
use defaultdict instead of try/except

Reply:
I used that initially, and it was slower than try/except. Not by much but I didn't want it to be slow :)

Reply:
Done